### ISSUE TITLE
Search input in the side-nav has no visible label

### ISSUE THREAD

**Issue by dakahn** [carbon-design-system/carbon-website#298]
### Steps to recreate
1) Run an aXe audit on the main page of the web site

There is in fact a label that's 1px by 1px wide and purposefully hidden and then the placeholder is used as a stand in. **The contrast for the placeholder text itself isn't up to WCAG spec.**

More broadly placeholders serving as labels in form fields of any kind is a design pattern we should move away from. [Here's some good reading on the reasons why](https://www.nngroup.com/articles/form-design-placeholders/).

## Solutions
There are few ways to address this. A good ol' fashion form label stuck right above the relevant box in [this style](https://www.carbondesignsystem.com/components/form/code) _or_ to stay stylistically consistent with what we have upping the color contrast on the placeholder text would improve accessibility greatly.

**Comment by joshblack** [comment]
_Issue grooming 1/18_

- Could be caused by algolia, requires some investigation to resolve

**Comment by dakahn** [comment]
Testing with February 2019 DAP ruleset and this is no longer an issue 👍🏽
